lib/mayaUsd/commands/PullPushCommands.cpp

```diff
@@ -452,9 +452,9 @@ MStatus DuplicateCommand::doIt(const MArgList& argList)
         OpUndoItemRecorder undoRecorder(_undoItemList);
 
         auto& manager = PXR_NS::PrimUpdaterManager::getInstance();
-        status = manager.duplicate(_srcPath, _dstPath, userArgs) ? MS::kSuccess : MS::kFailure;
+        auto  dstUfePaths = manager.duplicate(_srcPath, _dstPath, userArgs);
 
```

Comment:
Duplicate function now returns a list of destination paths. Allows getting rid of much code below.